docs/integration/simple-s3.md

```diff
@@ -18,7 +18,19 @@ composer require async-aws/simple-s3
 ```php
 use AsyncAws\SimpleS3\SimpleS3Client;
 
-$s3 = new SimpleS3Client();
+$s3 = new SimpleS3Client([
+    'accessKeyId' => 'your_access_key',
+    'accessKeySecret' => 'your_access_key_secret',
+    'region' => 'eu-central-1',
+]);
+
+$s3->createBucket([
+    'Bucket' => 'my-image-bucket',
+    'CreateBucketConfiguration' => [
+        'LocationConstraint' => $s3->getConfiguration()->get(Configuration::OPTION_REGION)
```

Comment:
When deploying on AWS, you usually dont specify the region in your config. Let's show both examples. 

```suggestion
        'LocationConstraint' => 'eu-central-1'
        //  'LocationConstraint' => $s3->getConfiguration()->get(Configuration::OPTION_REGION)
```